docs/notebooks/test_snia.ipynb

```diff
@@ -96,7 +96,8 @@
     "res, passbands = run_snia_end2end(\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        solid_angle=0.002,
</pre><p>Is it LSST's field of view in steradians? Could you please put a comment here.</p><p> </p><p>Also we use field-of-view radius when matching positions with opsim, should we use introduce these numbers as constants in the code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tdastro/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Random solid angle to get a reasonable number for testing. That sounds good to me to put field of view for LSST as constant.</p>